test/utils/duration.test.ts

```diff
@@ -0,0 +1,56 @@
+import { expect, test } from "@jest/globals";
+import { formatDuration } from "../../src/utils/duration";
```

Comment:
**nit:**

If you configured path alias:
```suggestion
import { formatDuration } from "@src/utils/duration";
```

Reply:
Wasn't configured, but I just configured it. Had to configure Jest to accept path aliases defined in tsconfig, so I also switched the Jest config from JS to TS, while I was at it.